frontend/src/components/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Typography, Box, Container } from '@mui/material';
+
+import Button from 'components/Button';
+
+const ListItem: React.FC = () => (
+  <Container
+    style={{
+      borderRadius: '15px',
+      border: '1px solid tomato',
+      // width: '500px',
+      // maxHeight: '600px',
+      display: 'flex',
+      flexDirection: 'column',
+      padding: '5px',
+    }}
```

Comment:
What is the role of ListItem, where do you use it? Could you make clear the component name? 
Also please move styles to styled.ts and remove all comments
Also data should be passed by props item, which takes an object with initial data instead of writing this inside the code. It will allow us to reuse the component

Reply:
Wep, I put it some days ago for test and forgot deleted this item. There is no use in project.

Reply:
Remove please

Reply:
Deleted as useless